src/containers/ec2.js

```diff
@@ -0,0 +1,44 @@
+const axios = require('axios');
+const utils = require('../utils');
+const eventIterface = require('../event');
+
+let currentEC2Labels = null;
+
+const URL = 'http://169.254.169.254/latest/meta-data/';
+const attributeToGet = ['instance-id', 'instance-type', 'local-ipv4', 'public-hostname', 'public-ipv4'];
+
+/**
+ * Load EC2 metadata and store it
+ * @returns {Promise} when resolved will contain the EC2 metadata
+ */
+module.exports.loadEC2Metadata = function loadEC2Metadata() {
+    if (currentEC2Labels) return Promise.resolve(currentEC2Labels);
+    const promises = [];
+    utils.debugLog('Loading EC2 metadata');
+    attributeToGet.forEach((attribute) => {
+        promises.push(axios.get(URL + attribute).then((response) => {
+            utils.debugLog(`Received response for ${attribute}: ${response.data}`);
+            if (response.status === 200) {
+                if (!currentEC2Labels) currentEC2Labels = {};
+                currentEC2Labels[`ec2.${attribute}`] = response.data;
```

Comment:
I think we should do `aws.ec2.attribute`

Reply:
You're right.